libs/platform/ebpf_serialize.c

```diff
@@ -6,7 +6,7 @@
 #include "ebpf_serialize.h"
 
 void
-ebpf_map_information_array_free(uint16_t map_count, _In_count_(map_count) ebpf_map_information_t* map_info)
+ebpf_map_information_array_free(_In_ uint16_t map_count, _In_count_(map_count) ebpf_map_information_t* map_info)
```

Comment:
What error was this hitting?   Both ebpf_serialize.c and h had no SAL annotation on this arg, and value types should not have any.

Reply:
This is the error that I am seeing.
```
 d:\a\ebpf-for-windows\ebpf-for-windows\libs\platform\ebpf_serialize.h(85): error C28218: For function 'ebpf_map_information_array_free' _Param_(2), for annotation 'SAL_null' the annotation's parameter 1 does not match that found at d:\a\ebpf-for-windows\ebpf-for-windows\libs\platform\ebpf_serialize.h(85). [D:\a\ebpf-for-windows\ebpf-for-windows\libs\platform\user\platform_user.vcxproj]
```

Reply:
Going to try removing them and see if it still repros.

Reply:
OK, looks like it passes with no SAL on the scalar value.